Tasks/AzureResourceManagerTemplateDeploymentV3/task.json

```diff
@@ -246,10 +246,11 @@
     {
       "target": "resourceGroupName",
       "endpointId": "$(ConnectedServiceName)",
-      "dataSourceName": "AzureResourceGroupsByExplicitSubscription",
-      "parameters": {
-        "subscriptionId": "$(subscriptionName)"
-      }
+      "endpointUrl": "{{{endpoint.url}}}/subscriptions/{{{endpoint.subscriptionId}}}/resourcegroups?$skiptoken={{{skiptoken}}}&api-version=2016-02-01",
+      "callbackContextTemplate": "{\"skiptoken\": \"{{#getTokenValue response.nextLink}}{{extractUrlQueryParameter %24skiptoken}}{{/getTokenValue}}\"}",
+      "callbackRequiredTemplate": "{{isTokenPresent response.nextLink}}",
+      "initialContextTemplate": "{\"skiptoken\": \"\"}",
+      "resultSelector": "jsonpath:$.value[*].name"
```

Comment:
Do we have other tasks with the same logic we should also cover with this change?

Reply:
In none of the task we have a pagination logic implemented for Resource group but we received ICM for this task and we believe it is due to pagination, Based on the customer response we may take it as repair item to implement for other task